tracer/build/_build/Build.Utilities.cs

```diff
@@ -356,6 +358,47 @@ private void ReplaceReceivedFilesInSnapshots()
         }
     }
 
+    private async Task<bool> IsDebugRun()
+    {
+        var forceDebugRun = Environment.GetEnvironmentVariable("ForceDebugRun");
+        if (!string.IsNullOrEmpty(forceDebugRun)
+         && (forceDebugRun == "1" || (bool.TryParse(forceDebugRun, out var force) && force)))
+        {
+            return true;
+        }
+
+        var buildId = Environment.GetEnvironmentVariable("BUILD_BUILDID");
+        if (string.IsNullOrEmpty(buildId))
+        {
+            // not in CI
+            return false;
+        }
+
+        try
+        {
+            var azDoApi = $"https://dev.azure.com/datadoghq/dd-trace-dotnet/_apis/build/builds/{buildId}";
+            using var client = new HttpClient();
+            using var stream = await client.GetStreamAsync(azDoApi + buildId);
```

Comment:
You have already set the buildId above, no?
```suggestion
            using var stream = await client.GetStreamAsync(azDoApi);
```

Reply:
Oops, too much copy-paste 😉  thanks!